ncm-metaconfig/src/main/metaconfig/haproxy/defaults.tt

```diff
@@ -0,0 +1,13 @@
+[% FOREACH config IN defaults.config.pairs -%]
+[% config.key %] [% config.value %]
+
+[%- END -%]
```

Comment:
it makes no sense to set an empty line and then on the next line do a `[%-` (the `-` cuts the previous whitespace character, in this case a newline). you can do the same with
```
[% config.key %] [% config.value %]
[% END %]
```